package.json

```diff
@@ -75,11 +75,13 @@
         "@openapi-contrib/openapi-schema-to-json-schema": "3.2.0",
         "ajv": "8.10.0",
         "ajv-formats": "2.1.1",
+        "buffer": "6.0.3",
```

Comment:
Why can't we use the built-in `Buffer` API in node.js? It should be for client applications to polyfill for the browser if it's needed.

Reply:
Addressed ✅

Reply:
Addressed ✅